frontend/plan/components/selector/TagList.tsx

```diff
@@ -26,7 +29,7 @@ export default function TagList({
                     className="taglist"
                     style={{ height: expanded ? "auto" : 0 }}
                 >
-                    {hiddenTags.map((elt) => (
+                    {hiddenTags.map((elt, i) => (
```

Comment:
what was the reason for adding i? do you need it in typescript?

Reply:
i actually think i don't need this here :P thanks for pointing it out!